File: maxmin.java
import java.util.*;
import java.lang.*;
import java.io.*;
class maxmin{
	public static void main(String[] args){
	Scanner in=new Scanner(System.in);
	String ar[]=in.nextLine().split(" ");
	int arr[]=Arrays.asList(ar).stream().mapToInt(Integer::parseInt).toArray();
	int sum=0;
	int min=Integer.MAX_VALUE;
	int max=Integer.MIN_VALUE;
	for(int i:arr){
	    sum=sum+i;
	    if(i>max) max=i;
	    if(i<min) min=i;
	}
	System.out.println((sum-max)+" "+(sum-min));
}
}

File: maxmin.py
n=int(input())
l=[int(i) for i in input().split()]
mx=max(l);mn=min(l)
l.remove(mx);l.remove(mn)
mx=-9999;mn=99999
for num in l:
    if(num>mx):
        mx=num
    if(num<mn):
        mn=num
print(mx,mn)